config/master.d/master.conf

```diff
@@ -3,7 +3,8 @@ auto_accept: True
 interface: 0.0.0.0
 event_return: mysql
 presence_events: True
-worker_threads: 10
+worker_threads: 16
+timeout: 180
```

Comment:
Right now we are using the `salt-api` for a lots of things on the UI (without background jobs). My main fear is that the request from the customer to velum might timeout even before it can report any error (this depends on a bunch on intermediate timeouts, but can happen).

5 seconds might sound a very low timeout, but when you are working with lots of minions it makes sense to work with such low timeouts. Also, reading the bug thread I'm confused, since it seemed that we were hitting the etcd discovery timeout (and some etcd instances failed to register against the discovery in time), not sure how increasing the salt timeout would help here.